examples/webgl_materials_sheen.html

```diff
@@ -0,0 +1,378 @@
+<html lang="en">
+	<head>
+		<title>Ammo.js softbody cloth demo</title>
```

Comment:
Can you please add this example to `files.js`?  Otherwise it won't be visible in the side bar.

Reply:
Done